src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
Probably because there's much to much ckde out there made unreadable by too many goto's, sometimes even leaving one function jumping into the middle of another *shudder*.
And concerning your age:
I teached myself programming on an "Amstrad CPC464" under "Locomotive BASIC", later Z80-Assembler and finaly K&R C (the last one under CP/M) on the same 'machine' ;)